kpops/component_handlers/kafka_connect/connect_wrapper.py

```diff
@@ -32,13 +31,14 @@ def __init__(self, host: str | None):
             )
             log.error(error_message)
             raise RuntimeError(error_message)
+        self._client = httpx.AsyncClient(base_url=host)
         self._host: str = host
```

Comment:
I believe this and the property aren't needed anymore

Reply:
it is needed for debugging